src/elm/View.elm

```diff
@@ -44,3 +47,13 @@ resourcesList resources =
             )
             resources
         )
+
+
+viewEvent : Expirable Event -> Html a
+viewEvent expirable =
+    case Expirable.value expirable of
+        GlobalRateIncrease ->
+            div [] [ text "Increase income rate globally" ]
```

Comment:
is the plan here to use an onChange click handler to create an expire-able that increases global rates?  Where will those live in the model record?

Reply:
Something like that. I was thinking maybe having a button with an `onClick` handler. That will add an expire-able multiplier to the inventory (which itself is a key on the main model).